product/personas/Danielle_Developer.md

```diff
@@ -0,0 +1,37 @@
+# Danielle, the Developer
+
+<img title="" src="https://lh3.googleusercontent.com/EshCB9N2Nci8huAH8HgthX_J7qJ7y_etMX_5gdR0L99wbA4Q_gI4Z6KyJhipzhVsaI5MWR6R9ALmisXWMXRe4tXCvsTuUhieA-HMMnguTm8Hzi2gNsLi9TYLvYRS7pIlXlStERUm09mpoDqWJueUR8LSlAUzXHFYlnD5AyuRNuFUIlCaC7KHWKz0bTcAvA" alt="" data-align="left" width="198">
+
+Age: 24
+
+Job Level: Individual Contributor
+
+Software Skillset: Computer Science & AI, BSc
+
+### Job To Be Done:
+
+- Software Developer in New Startup Inc. (works for [Sahib](./Sahib Startup CTO.md))
```

Comment:
Does this link work? It's missing underscores

Reply:
There are a few technical issues with the way they're written, some clean up is required, e.g. images don't render as they are at the moment either. Plan on sweeping through and fixing these issues once we have consistent personas in Figma.